internal/k8s/env.go

```diff
@@ -54,3 +65,29 @@ func EnvFromString(s string) Env {
 	}
 	return EnvUnknown
 }
+
+func EnvFromConfig(config *api.Config) Env {
+	n := config.CurrentContext
+
+	c, ok := config.Contexts[n]
+	if !ok {
+		return EnvUnknown
+	}
+
+	cn := c.Cluster
```

Comment:
should we also add Cluster to `tilt doctor`?

Reply:
Yeah good idea